### DISCUSSION TITLE
possible to add page size?

### DISCUSSION THREAD

**Discussion by ralyodio** [searx/searx#2505] [General]
I'm only getting 30 results back by default. Is it possible to change it to 100?

https://searx.envs.net/search?category_general=1&q=svelte+js&time_range=week&language=en-US&format=json

**Comment by unixfox** [comment]
It's not searx that result 30 results but the engine used. Searx cannot do much about that unfortunately.

**Reply by ralyodio** [reply]
i see. so i would have to run  my own instance? Because I saw a setting for "page size" in the config.

**Reply by unixfox** [reply]
> i see. so i would have to run my own instance? Because I saw a setting for "page size" in the config.

I'm not entirely sure what's this setting doing. It could help for having more results per page or could just be a setting to ask the engine for more results per page.
